src/internal/utils.js

```diff
@@ -134,7 +134,7 @@ const kThrow = err => {
 }
 const kReturn = value => ({ value, done: true })
 export function makeIterator(next, thro = kThrow, name = 'iterator') {
-  const iterator = { name, next, throw: thro, return: kReturn }
+  const iterator = { meta: { name }, next, throw: thro, return: kReturn, isSagaIterator: true }
```

Comment:
had to wrap in `meta`  in sake of consistency, probably it's better to flatten `meta` completely (switch back to name & location properties)